src/pages/backmystack.js

```diff
@@ -0,0 +1,119 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+
+export default class BackMyStack extends React.Component {
+  static getInitialProps({ req, query }) {
+    const uuid = query.uuid; // handle case where the uuid is not supplied
+    const opencollectiveBaseUrl = process.env.OPENCOLLECTIVE_BASE_URL;
```

Comment:
The convention accross our projects it to name this variable `WEBSITE_URL`. Maybe, we can name it `OPENCOLLECTIVE_WEBSITE_URL` here.